### PR TITLE
feat: dockerise stack 

### PR DESCRIPTION
**Pull request by edisontim**
This PR introduces dockerization of the stack except for client so: `katana`, `torii` and `lore-machine` are all running in a contained environment. Instructions to run the dockers are present in the root readme file

### CODE REVIEW

**Review by cwastche — COMMENTED** [review]
Looks perfect to me. It works on my end.

File: client/package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "react-dom": "^18.2.0",
     "react-draggable": "^4.4.5",
     "react-popper": "^2.3.0",
+    "react-use-websocket": "latest",
```
Comment (jobez) [inline client/package.json:45]: do we want to pin to latest? might be good to use the specific version that is currenty working to spare us from headache if updates produce breaking changes

File: contracts/scripts/torii.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+set_config() {
+	source ./scripts/env_variables.sh dev
+	export STARKNET_RPC_URL=http://katana:5050
+	printf '0.1\n' | ./scripts/set_config.sh
+}
+
+sozo migrate --rpc-url http://katana:5050
+
+# TODO: Get world address from migrate
```
Comment (jobez) [inline contracts/scripts/torii.sh:12]: is this TODO represented somewhere? github issue would be good.

File: package.json [hunk 3]
```diff
@@ -10,4 +10,4 @@
     "build-client": "bun run --cwd ./client build"
   },
   "dependencies": {}
-}
\ No newline at end of file
+}
```
Comment (jobez) [inline package.json:13]: unnecessary newline?

**Review by jobez — COMMENTED** [review]
smol comments

File: package.json (hunk 3 above)
Comment (edisontim) [reply package.json:13]: What formatter do you use? Mine's adding it

File: client/package.json (hunk 1 above)
Comment (edisontim) [reply client/package.json]: Yeah you're right, changed that

File: package.json (hunk 3 above)
Comment (jobez) [reply package.json]: kk, if its a formatting thing then we keep

**Review by jobez — APPROVED** [review]
lgtm

**Pull request merged**